src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -22,11 +25,12 @@
     /**
      * Constructs a {@code CommandResult} with the specified fields.
      */
-    public CommandResult(boolean userUpdated, String dataToUser, String feedbackToUser,
-                         boolean showHelp, boolean exit) {
+    public CommandResult(String dataToUser, String feedbackToUser, boolean userUpdated,
```

Comment:
When we call `new CommandResult`, it becomes increasingly hard to track which boolean means what. I'm thinking how we can possibly improve on this.. 

Do you think it would be a good idea to abstract the variables first? 

So instead of:
```java
new CommandResult("This is dataToUser", "This is feedbackToUser", true, false, false, false);
```

It could be something like:
```java
boolean userUpdated = true;
boolean showPreview = false;
boolean showHelp = false;
boolean exit = false;
new CommandResult("This is dataToUser", "This is feedbackToUser", userUpdated, showPreview, showHelp, exit);
```

Reply:
Great suggestion, I'm on it.

Reply:
Actually will wait on more comments hahaha